cognite/neat/_session/_mapping.py

```diff
@@ -37,3 +53,81 @@ def classic_to_core(self, org_name: str) -> None:
         )
 
         self._state.data_model.write(output.rules, change)
+
+        start = datetime.now(timezone.utc)
+
+        source_id, rules = self._state.data_model.last_verified_dms_rules
```

Comment:
Note now that there are three transformations that are done by this method
1. Mapping
2. Including all referenced views and containers.
3. Renaming to parent name.

Will move this in a refactoring.